neuro_san/service/watcher/registries/polling_registry_observer.py

```diff
@@ -14,10 +14,11 @@
 from pathlib import Path
 from watchdog.observers.polling import PollingObserver
 
-from neuro_san.service.registries_watcher.periodic_updater.registry_change_handler import RegistryChangeHandler
+from neuro_san.service.watcher.registries.registry_change_handler import RegistryChangeHandler
+from neuro_san.service.watcher.registries.registry_observer import RegistryObserver
 
 
-class RegistryPollingObserver:
+class PollingRegistryObserver(RegistryObserver):
```

Comment:
Rename this class to better match the naming convention when we derive from something else.